lib/update-globals.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
```

Comment:
Let's put this in a `scripts/` directory. Then when publishing to npm (or, more likely in our case, importing into the Chromium tree) we can avoid including that directory.

The output should still be in `lib/` though.

Reply:
I added `scripts/*` to the `.gitignore`, I'm assuming we want that

Reply:
Oh I did this backwards, let me fix now